app/frontend/src/features/messages/groupchats/GroupChatsTab.tsx

```diff
@@ -52,7 +52,7 @@ export default function GroupChatsTab() {
   }
 
   useEffect(() => {
-    fetchGroupChats();
+    void fetchGroupChats();
```

Comment:
What difference does this make if we don't need to await on this?

Reply:
I hope it does not make any difference for the user. But do you prefer to await?

Reply:
I added the `void` because of an eslint warning.

Reply:
Oh that's strange, I didn't get a lint warning on this. But I'm ok with leaving it in, makes it a bit clearer it returns a value but we are not using it in this case